borg/archiver.py

```diff
@@ -936,6 +937,10 @@ def build_parser(self, args=None, prog=None):
         subparser.add_argument('-n', '--dry-run', dest='dry_run',
                                default=False, action='store_true',
                                help='do not change repository')
+        subparser.add_argument('-i', '--inplace', dest='inplace',
+                               default=False, action='store_true',
+                               help="""rewrite repository in-place, with no chance of going back to older
```

Comment:
google doesn't seem to distinguish in its search results, giving both the definition of "in place". oddly enough, "in-place" yields 1000 more search results, however. the free dictionnary puts it without a dash: http://idioms.thefreedictionary.com/in+place